Fix tests to find the dynamic library in the build tree. In the latest
unreleased upstream code the executable has been changed to link with
the static library. It has been proposed that this should be changed
back: https://github.com/waywardgeek/sonic/issues/16
If they do not, then this patch will no longer be needed.
--- Makefile.orig	2021-08-17 18:00:21.000000000 -0500
+++ Makefile	2021-08-17 18:11:30.000000000 -0500
@@ -72,4 +72,4 @@
 	rm -f *.o sonic $(LIB_NAME)* libsonic.a test.wav
 
 check:
-	./sonic -s 2.0 ./samples/talking.wav ./test.wav
+	DYLD_LIBRARY_PATH=. ./sonic -s 2.0 ./samples/talking.wav ./test.wav
